src/app/components/reactive-forms.compnent.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { ReactiveFormsModule, FormsModule } from "@angular/forms";
+import { ReactiveFormsComponent, User } from "./reactive-forms.component";
+
+
+describe('Component: ReactiveForms', () => {
+
+    let component: ReactiveFormsComponent;
+    let fixture: ComponentFixture<ReactiveFormsComponent>;
+
+    beforeEach(() => {
+
+        // refine the test module by declaring the test component
+        TestBed.configureTestingModule({
+            imports: [ReactiveFormsModule, FormsModule],
+            declarations: [ReactiveFormsComponent]
+        });
+
+        // create component and test fixture
+        fixture = TestBed.createComponent(ReactiveFormsComponent);
+
+        // get test component from the fixture
+        component = fixture.componentInstance;
+        component.ngOnInit();
+    });
+
+    it('form invalid when empty', () => {
+        expect(component.form.valid).toBeFalsy();
+    });
+
+    it('name field validity', () => {
+        let errors = {};
+        let name = component.form.controls['name'];
```

Comment:
I'd use `component.form.get('name')` rather than going over the `.controls['name']`. That's usually the way you access form controls on reactive Angular forms. Could you change this in the test?

Reply:
Yeah sure. Updated it using `get` method usage. Thanks!